### PR TITLE
[4.7.7] starting measurements automatically - Cleanups (patch 3/3)

### PR DESCRIPTION
**Pull request by vitalii-komisarenko**
Because of its size PR #1139 needs to be split.
Description: #1139 (review)

This PR reflects the cleanup phase of the change.

This PR depends on #1290 and #1289 in order to work properly.

### CODE REVIEW

**Comment by tomereli** [comment]
Please name the PRs so we will know which one should be reviewed first:
For example:
[4.7.7] starting measurements automatically - BWL (patch 1/3)
[4.7.7] starting measurements automatically - Monitor/Slave (patch 2/3)
[4.7.7] starting measurements automatically - Cleanups (patch 3/3)

File: agent/src/beerocks/monitor/monitor_thread.cpp [hunk 1]
```diff
@@ -772,64 +772,6 @@ bool monitor_thread::handle_cmdu_vs_message(Socket *sd, ieee1905_1::CmduMessageR
         received_error_notification_ack_retry = 0;
         break;
     }
-    case beerocks_message::ACTION_MONITOR_CLIENT_START_MONITORING_REQUEST: {
```
Comment (tomereli) [inline agent/src/beerocks/monitor/monitor_thread.cpp]: This has to be tested on RDKB/UGW - since UTAF is not yet ready I suggest @kantera800 you test it.

**Review by tomereli — COMMENTED** [review]
This PR is a bit more risky, it removes the VS path so has to be tested on RDKB and UGW by Intel team to make sure it doesn't break anything, so not approving **yet**.
@morantr @kantera800 please provide a version for @amritak23 or @Mila-Sh and ask for a QA test.
We can wait a bit with this PR but we do want to merge it in a couple of days max.

File: agent/src/beerocks/monitor/monitor_thread.cpp (hunk 1 above)
Comment (kantera800) [reply agent/src/beerocks/monitor/monitor_thread.cpp]: @tomereli @vitalii-komisarenko 
This PR is currently fails to build on RDKB/UGW platform so i cant test it.

_/home/akanterx/work/dev2/rdkb/atom_rdkbos/build/tmp/work/core2-32-rdk-linux/prplmesh/git+f8786935bdc0b58021710248bf2620896b74c44f-r0/git/agent/src/beerocks/monitor/monitor_thread.cpp:789:66: error: 'beerocks_message::sClientMonitoringParams {aka struct beerocks_message::sClientMonitoringParams}' has no member named 'vap_id'
          int vap_id                       = int(request->params().vap_id);_

Comment (morantr) [reply agent/src/beerocks/monitor/monitor_thread.cpp]: I was able to build it successfully on RDKB and UGW.

**Review by morantr — APPROVED** [review]

**Review by kantera800 — APPROVED** [review]
This PR  fixes start monitoring .
 Also,  @moran tested it on UGW/RDKB so approving.

**Pull request merged**